api/src/main/java/org/openmrs/module/openconceptlab/OclConceptService.java

```diff
@@ -17,7 +15,6 @@
 	 *  one couldn't be found.
 	 */
 	@Transactional(readOnly = true)
-	@Authorized(PrivilegeConstants.VIEW_CONCEPTS)
```

Comment:
@ibacher I had to remove this Authorized annotation, as this privilege is no longer in 2.x.  It's unfortunate we didn't just keep the constant as-is and change the value of it.